docs/search/search-query-language/search-operators/macro.md

```diff
@@ -0,0 +1,55 @@
+---
+id: macro
+title: macro Operator
+sidebar_label: macro
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+With `macro` operator you can reuse the portions of search query language in multiple other queries. This operator also includes arguments and also performs its respective evaluation of the arguments.
+
+Use backquote (``) character to refer macro operator in the query. Macros referred inside another macro is called **Nested macros**. Macro referred within the context of another macro query of is called **Inner Macro**. The macro where the inner macro is referred is called the **Outer Macro**.
+
+:::note
+- Only **Administrators** and **Users** with access to **Query Reference** will be able to run queries using macros.
```

Comment:
```suggestion
- Only **Administrators** and **Users** with access to **Query Reference** can run queries using macros.
```

Reply:
This should be **Macros** not query reference?

Reply:
I got this information from the macro operator confluence page. Please let me know if it is right to add it here?
![Uploading Screenshot 2024-11-28 at 11.22.01 AM.png…]()